### PR TITLE
add new extensions category and improve category assignments

### PR DESCRIPTION
**Pull request by Geminii**
Currently, some module aren't really Devtools module that help you only in dev, and things that just extend the capabilities of Nuxt. They aren't really dev-centric.
The proposed solution is to add a new category called `Extensions` to distinguish both. 

- [x] waiting for default extensions icon

Devtools category will contain only : eslint, typescript, typescript-runtime, webpack-profile, ngrok, webpackmonitor, localtunnel, typed-router, bundle-buddy, webpackdashboard.

Do you think we need to add another module into `Devtools category` like `python` or others for example ? 
@pi0

### CODE REVIEW

**Comment by pi0** [comment]
Nice work @Geminii 👍  Modules like python might be moved to misc/miscellaneous category probably? Also shall we merge with devtool icon or you prefer waiting for icon?

**Comment by Geminii** [comment]
> Nice work @Geminii 👍 Modules like python might be moved to misc/miscellaneous category probably? Also shall we merge with devtool icon or you prefer waiting for icon?

It might be wise to create a new miscellaneous category. Do you think that there would also be other modules that could fall into this category ? (maybe create a new PR for this one)

I would like to wait the icon because i know that `rmoon` will do a great job about it ❤️

**Comment by pi0** [comment]
> It might be wise to create a new miscellaneous category. Do you think that there would also be other modules that could fall into this category ? (maybe create a new PR for this one)

Actually python is only one i can think of as of now... Yet we need a cat for it 

> I would like to wait the icon because i know that rmoon will do a great job about it ❤️

That would be awesome then! Just didn't want to stall PR :)

File: modules/emotion.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ icon: emotion.png
 github: 'https://github.com/nuxt-community/emotion-module'
 website: 'https://github.com/nuxt-community/emotion-module'
 learn_more: 'https://emotion.sh/'
-category: Devtools
+category: Extensions
```
Comment (Atinux) [inline modules/emotion.yml:9]: I think this should be moved to UI

File: modules/leaflet.yml [hunk 2]
```diff
@@ -6,7 +6,7 @@ icon: leaflet.png
 github: 'https://github.com/schlunsen/nuxt-leaflet'
 website: 'https://github.com/schlunsen/nuxt-leaflet'
 learn_more: 'https://vue2-leaflet.netlify.app/quickstart/'
-category: Devtools
+category: Extensions
```
Comment (Atinux) [inline modules/leaflet.yml:9]: I think it may be moved to `Librairies`

File: modules/rfg-icon.yml [hunk 3]
```diff
@@ -6,7 +6,7 @@ icon: real-favicon-generator.png
 github: 'https://github.com/pimlie/nuxt-rfg-icon'
 website: 'https://github.com/pimlie/nuxt-rfg-icon'
 learn_more: 'https://realfavicongenerator.net/'
-category: Devtools
+category: Extensions
```
Comment (Atinux) [inline modules/rfg-icon.yml:9]: Shall we move this to `UI` since it is related to visual changes? cc @pi0

File: modules/stories.yml [hunk 4]
```diff
@@ -6,7 +6,7 @@ icon: ''
 github: 'https://github.com/richardeschloss/nuxt-stories'
 website: 'https://github.com/richardeschloss/nuxt-stories'
 learn_more: ''
-category: Devtools
+category: Extensions
```
Comment (Atinux) [inline modules/stories.yml:9]: I consider it as a Devtools or UI for this one.

File: modules/storybook.yml [hunk 5]
```diff
@@ -8,7 +8,7 @@ icon: storybook.png
 github: 'https://github.com/nuxt-community/storybook'
 website: 'https://storybook.nuxtjs.org'
 learn_more: 'https://storybook.js.org/'
-category: Devtools
+category: Extensions
```
Comment (Atinux) [inline modules/storybook.yml:11]: I consider it as a Devtools or UI for this one.

File: modules/svg-sprite.yml [hunk 6]
```diff
@@ -6,7 +6,7 @@ icon: ''
 github: 'https://github.com/nuxt-community/svg-sprite-module'
 website: 'https://github.com/nuxt-community/svg-sprite-module'
 learn_more: ''
-category: Devtools
+category: Extensions
```
Comment (Atinux) [inline modules/svg-sprite.yml:9]: Shall we move it to `UI`?

**Comment by Atinux** [comment]
I believe we should merge Librairies and Extensions together and use the `Librairies` icon :)

**Comment by Geminii** [comment]
> I believe we should merge Librairies and Extensions together and use the `Librairies` icon :)

It's a good alternative to create a new `Extensions` category. 
But about `python`, consider it as `Libraries` too ? or create a new `Misc` category ?

File: modules/rfg-icon.yml (hunk 3 above)
Comment (pi0) [reply modules/rfg-icon.yml:9]: SEO may also make sense (since PWA is there too)

File: modules/stories.yml (hunk 4 above)
Comment (pi0) [reply modules/stories.yml:9]: By considering "Extensions" as those ones add a change to protuction, keeping in Devtools makes sense

File: modules/storybook.yml (hunk 5 above)
Comment (pi0) [reply modules/storybook.yml:11]: (same for Keeping in devTools)

**Comment by pi0** [comment]
@Geminii If only python is problematic, you may remove it since it is an unmaintained/experimental module anyway

**Comment by Geminii** [comment]
Thanks for all of your great feedbacks. 
I will integrate them 😀

File: modules/rfg-icon.yml (hunk 3 above)
Comment (Atinux) [reply modules/rfg-icon.yml:9]: Agree!

**Comment by pi0** [comment]
> renaming Librairies to Extensions

Current libraries are modules that integrate an exisisting Vue/JS library with Nuxt while extensions like nuxt story adding a new feature to nuxt DX / Build system.

File: modules/emotion.yml (hunk 1 above)
Comment (Atinux) [inline modules/emotion.yml:9]:
```suggestion
category: UI
```

File: modules/leaflet.yml (hunk 2 above)
Comment (Atinux) [inline modules/leaflet.yml:9]:
```suggestion
category: Librairies
```

File: modules/rfg-icon.yml (hunk 3 above)
Comment (Atinux) [inline modules/rfg-icon.yml:9]:
```suggestion
category: SEO
```

File: modules/stories.yml (hunk 4 above)
Comment (Atinux) [inline modules/stories.yml:9]:
```suggestion
category: Devtools
```

File: modules/storybook.yml (hunk 5 above)
Comment (Atinux) [inline modules/storybook.yml:11]:
```suggestion
category: Devtools
```

File: modules/svg-sprite.yml (hunk 6 above)
Comment (Atinux) [inline modules/svg-sprite.yml:9]:
```suggestion
category: UI
```

**Comment by Atinux** [comment]
Updated the PR, it looks good for me: https://modules-of7rfl989.vercel.app/?orderBy=desc#Devtools

One note, should be move Storybook and Nuxt Stories to UI @pi0 ?

**Comment by pi0** [comment]
Technically storybook is not a nuxt module :D BTW i think being in devtools at least currently makes more sense since UI section is used for UI frameworks or modules that add UI components. Storybook while can be used for production, is mainly a development tool...

File: modules/emotion.yml [hunk 7]
```diff
@@ -6,7 +6,7 @@ icon: emotion.png
 github: 'https://github.com/nuxt-community/emotion-module'
 website: 'https://github.com/nuxt-community/emotion-module'
 learn_more: 'https://emotion.sh/'
-category: Extensions
+category: UI
```
Comment (pi0) [inline modules/emotion.yml:9]: I think emotion can be more an extension (adding build ability for css-in-js)

Comment (Atinux) [inline modules/emotion.yml:9]:
```suggestion
category: Extensions
```

**Comment by Atinux** [comment]
Should be perfect now, Vincent will make a logo when he will have time, no need to wait for merging this PR I think :)

**Review by pi0 — APPROVED** [review]

File: modules/localforage.yml [hunk 8]
```diff
@@ -6,7 +6,7 @@ icon: ''
 github: 'https://github.com/nuxt-community/localforage-module'
 website: 'https://github.com/nuxt-community/localforage-module'
 learn_more: 'https://localforage.github.io/localForage/'
-category: Devtools
+category: Extensions
```
Comment (pi0) [inline modules/localforage.yml:9]: This can be a library too

Comment (Atinux) [inline modules/localforage.yml:9]:
```suggestion
category: Librairies
```

**Pull request merged**